ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5861,6 +5861,8 @@ HTWHLSTC:
   - high_supply_air_temperature_alarm
   - low_return_air_temperature_alarm
   - low_supply_air_temperature_alarm
+  - heating_percentage_command
```

Comment:
I'm not sure I understand how these are cooling/heating commands given to the heat wheel. The heat wheel can only modulate its rotational speed to transfer more or less energy between the supply and exhaust air streams. What do these fields mean in this context?

Reply:
Hi @cstirdivant - apologies for confusion. The heating and cooling commands are actually coils. These are a separate unit to the the heatwheel. The thermal wheel will heat supply air to a certain amount then the chilling/heating valves operate. I have changed the HTWHLSTC abstract type back and submitted the DOAS entity type. Thanks, Ciaran